rlhive/envs.py

```diff
@@ -193,7 +196,7 @@ def register_myo_envs():
                 env,
                 variants={
                     "obs_keys": [
-                        "hand_jnt",
+                        "qpos", # TODO: Check if this is correct
```

Comment:
@ShahRutav  good catch on this. 
The `hand_jnt` doesn't exist. `qpos` seems like the right choice. 

Ideally, this should have all DoFs of the agent but not the object. Since there is no object in these scene `qpos` corresponds to the right thing.